Formula/srcml.rb

```diff
@@ -0,0 +1,194 @@
+class Srcml < Formula
+  desc "convert source code to XML"
+  homepage "http://www.srcml.org/"
+  url "http://sead1.open.ac.uk/srcml/srcml-0.9.5.tar.gz"
+  version "0.9.5"
+  sha256 "55dd2115548e270724af4251187343656d2dfda0e7d372fee15ae27262e3fa8e"
+
+  depends_on "cmake" => :build
+  depends_on "LibArchive" => :build
```

Comment:
In a previous review round, PR author said srcml needs a newer version than macOS provides.